tests/rptest/clients/rpk.py

```diff
@@ -573,71 +573,11 @@ def group_seek_to(self, group, to):
-
-        partition_pattern_static_member = re.compile(
-            "(?P<topic>[^\s]+) +(?P<partition>\d+) +(?P<offset>\d+|-) +(?P<log_end>\d+|-) +(?P<lag>-?\d+|-) *(?P<member_id>[^\s]*)? *(?P<instance_id>[^\s]*) *(?P<client_id>[^\s]*)? *(?P<host>[^\s]*)?"
-        )
-        partition_pattern_dynamic_member = re.compile(
-            "(?P<topic>[^\s]+) +(?P<partition>\d+) +(?P<offset>\d+|-) +(?P<log_end>\d+|-) +(?P<lag>-?\d+|-) *(?P<member_id>[^\s]*)? *(?P<client_id>[^\s]*)? *(?P<host>[^\s]*)?"
-        )
-
-        def check_lines(lines):
-            for line in lines:
-                # UNKNOWN_TOPIC_OR_PARTITION: This server doesn't contain this partition or topic.
-                # We should wait until server will get information about it.
-                if line.find('UNKNOWN_TOPIC_OR_PARTITION') != -1:
-                    return False
-
-                # Leadership movements are underway
-                if 'NOT_LEADER_FOR_PARTITION' in line:
-                    return False
-
-                # Cluster not ready yet
-                if 'unknown broker' in line:
-                    return False
-
-                if "missing from list offsets" in line:
-                    return False
-
-            return True
-
-        def parse_partition(string):
-
-            pattern = partition_pattern_dynamic_member
-            if static_member_pattern.match(string):
-                pattern = partition_pattern_static_member
-            m = pattern.match(string)
-
-            # Check to see if info for the partition was queried during a change in leadership.
-            # if it was we'd expect to see a partition string ending in;
-            # NOT_LEADER_FOR_PARTITION: This server is not the leader for that topic-partition.
-            if m is None and string.find('NOT_LEADER_FOR_PARTITION') != -1:
-                return None
-
-            # Account for negative numbers and '-' value
-            all_digits = lambda x: x.lstrip('-').isdigit()
-
-            offset = int(m['offset']) if all_digits(m['offset']) else None
-            log_end = int(m['log_end']) if all_digits(m['log_end']) else None
-            lag = int(m['lag']) if all_digits(m['lag']) else None
-
-            return RpkGroupPartition(topic=m['topic'],
-                                     partition=int(m['partition']),
-                                     current_offset=offset,
-                                     log_end_offset=log_end,
-                                     lag=lag,
-                                     member_id=m['member_id'],
-                                     instance_id=m.groupdict().get(
-                                         'instance_id', None),
-                                     client_id=m['client_id'],
-                                     host=m['host'])
+        def try_parse_field(l: str):
+            m = re.compile(f"^\s*(?P<name>[^\s]+) +(?P<value>[^\s]+)$").match(
```

Comment:
nit: there isn't much value in compiling a regexp and abandoning it; either make it a global var or use re.match(...) since performance doesn't matter there